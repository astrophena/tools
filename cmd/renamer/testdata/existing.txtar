-- 1.txt --
lol
-- 2.txt --
lol2
-- 3.txt --
lol3
-- 99.txt --
lol4
-- 10.txt --
lol5
