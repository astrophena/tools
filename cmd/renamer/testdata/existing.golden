-- 1.txt --
lol
-- 10.txt --
lol5
-- 2.txt --
lol2
-- 4.txt --
lol3
-- 5.txt --
lol4
